packages/vue/src/components/search/DataSearch.jsx

```diff
@@ -504,12 +550,12 @@ const DataSearch = {
 		},
 		handleVoiceResults({ results }) {
 			if (
-				results
-				&& results[0]
-				&& results[0].isFinal
-				&& results[0][0]
-				&& results[0][0].transcript
-				&& results[0][0].transcript.trim()
+				results &&
```

Comment:
@mohdashraf010897 Why formatting changes? Are you using some different prettier config?